src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/watchdog/SpeedHypixelBHop.kt

```diff
@@ -46,15 +46,9 @@ object SpeedHypixelBHop : Choice("HypixelBHop") {
     override val parent: ChoiceConfigurable<Choice>
         get() = ModuleSpeed.modes
 
-    private val horizontalAcceleration by boolean("HorizontalAcceleration", true)
-    private val verticalAcceleration by boolean("VerticalAcceleration", true)
-
-    private const val BASE_HORIZONTAL_MODIFIER = 0.0004
-
-    // todo: check if we can do more with this
-    private const val HORIZONTAL_SPEED_AMPLIFIER = 0.0007
-    private const val VERTICAL_SPEED_AMPLIFIER = 0.0004
-
+    private val horizontalAcceleration by float("HorizontalAcceleration", 0.00F, 0.00F..1.00F)
+    private val verticalAcceleration by float("VerticalAcceleration", 0.01F, 0.00F..1.00F)
```

Comment:
There is a problem. It's impossible to set such precise values (like 0.0007 or  0.0004) with those values represented as sliders in the GUI. Users will probably be in a situation where 0.00 is too little and 0.01 is too much.

Reply:
I have thought about that..

The best course of action would probably be to make an initial value for the horizontal (and vertical) speed modifier that can simply be added onto with the slider, but that would render this speed useless for every other server apart from hypixel due to the vertical modification

Reply:
Or I could just increase the scale of the horizontal acceleration and then divide it when it’s used

Reply:
I will resolve this by simply making the slider with lower bounds and by multiplying the initial horizontal acceleration